sdk/python/sdk/zrok/setup.py

```diff
@@ -1,16 +1,20 @@
-from setuptools import setup, find_packages  # noqa: H301
 import os
 
-NAME = "zrok_sdk"
-VERSION = "0.4.0.dev"
-try:
-    VERSION = os.environ['ZROK_VERSION']
-except KeyError: 
-    pass
+from setuptools import find_packages, setup  # noqa: H301
+
+# optionally upload to TestPyPi with alternative name in testing repo
+NAME = os.getenv('ZROK_PY_NAME', "zrok_sdk")
```

Comment:
Use the new var if defined, else use the same name `zrok_sdk` (normalizes to `zrok-sdk` in PyPi)